qiskit/circuit/quantumcircuitdata.py

```diff
@@ -16,15 +16,13 @@
 from collections.abc import MutableSequence
 
 import qiskit._accelerate.circuit
+from qiskit._accelerate.circuit import CircuitInstruction  # pylint: disable=unused-import
```

Comment:
Technically I think the right way to do this is to add an `__all__` variable, and include `"CircuitInstruction"` in it (along with the other things that should be there, which I think is just `"QuantumCircuitData"`) - that marks it as "explicitly re-exported".

Reply:
I made a mistake, it was actually the preceding import that was unused, and I simply removed it.